website/static/js/profile.js

```diff
@@ -207,6 +209,15 @@ var BaseViewModel = function(urls, modes, preventUnsaved) {
     // Must be set after isValid is defined in inherited view models
     self.hasValidProperty = ko.observable(false);
 
+    self.savingText = ko.computed(function() {
```

Comment:
This seems to shadow the declaration of `self.savingText()` on [Line 198](https://github.com/CenterForOpenScience/osf.io/pull/3941/files#diff-9cd61d5035427d03e2bdefab7288fd55R198).